src/sql/src/names.rs

```diff
@@ -2351,6 +2352,70 @@ where
     ResolvedIds::new(visitor.ids)
 }
 
+#[derive(Debug)]
+pub struct RawItemDependencyIds {
+    pub ids: BTreeSet<GlobalId>,
+}
+
+impl<'ast> Visit<'ast, Raw> for RawItemDependencyIds {
+    fn visit_item_name(&mut self, item_name: &RawItemName) {
+        if let RawItemName::Id(id, _, _) = item_name {
+            let parsed_id = id.parse::<GlobalId>().unwrap();
+            self.ids.insert(parsed_id);
+        }
```

Comment:
I'm skeptical that this is correct, because it will skip over all named references leading to a potentially incorrect order.